### PR TITLE
[#18] add glitter 404 template

### PR DESCRIPTION
**Pull request by LeidiFlores**
## Added Glitter theme! 

image

### CODE REVIEW

**Review by matteobruni — CHANGES_REQUESTED** [review]
Please read carefully #18, you didn't update a required file.

**Comment by LeidiFlores** [comment]
I already added it, please check again @matteobruni

**Review by matteobruni — CHANGES_REQUESTED** [review]
Please check the theme using the main index.html, is not working on this PR.

**Comment by LeidiFlores** [comment]
sorry, I missed a comma, it's ready now :)

File: glitter/js/404.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            speed: 0.17,
+            direction: "none",
+            random: true,
+            straight: false,
+            outModes: {
+                default: "out"
+            },
+        }
+    },
+    interactivity: {
```
Comment (matteobruni) [inline glitter/js/404.js:48]: This `interactivity` section is useless, please remove it.

**Review by matteobruni — CHANGES_REQUESTED** [review]
This configuration is really similar to the `ocean` one: https://tsparticles.github.io/404-templates/ocean/404.html

Please make it different, the noticeable difference is the shape type, but I don't think is enough to consider it different.

**Comment by LeidiFlores** [comment]
Ok, @matteobruni let me check

**Comment by LeidiFlores** [comment]
hi @matteobruni ! 
I have a lot of problems with this MR, I'll close it and create another one, can I?

**Comment by matteobruni** [comment]
> hi @matteobruni !
> I have a lot of problems with this MR, I'll close it and create another one, can I?

Yes no problem

**Pull request closed without merging**